app/uk/gov/hmrc/gatekeeper/utils/SortingHelper.scala

```diff
@@ -35,11 +35,11 @@ object SortingHelper {
   }
 
   def descendingVersionWithoutFields(v1: VersionSubscriptionWithoutFields, v2: VersionSubscriptionWithoutFields) = {
```

Comment:
The library provides a sort for version nbr that might be better tested and applicable here.

Reply:
Although this is descending so 🤷

Reply:
Used the sorting provided in the library - see SubscriptionController.